caikit_computer_vision/modules/text_to_image/sdxl_stub.py

```diff
@@ -0,0 +1,76 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for text to image via SDXL.
+"""
+# Standard
+from typing import Union, get_args
+import os
+
+# Third Party
+import numpy as np
+
+# First Party
+from caikit.core.modules import ModuleBase, ModuleConfig, ModuleSaver, module
+from caikit.interfaces.vision import data_model as caikit_dm
+import alog
+
+# Local
+from ...data_model import TextToImageResult
+from ...data_model.tasks import TextToImageTask
+
+log = alog.use_channel("SDXL")
+
+
+@module(
+    id="28aa938b-1a33-11a0-11a3-bb9c3b1cbb11",
+    name="Stub module for Text to Image",
+    version="0.1.0",
+    task=TextToImageTask,
+)
+class SDXLStub(ModuleBase):
```

Comment:
It looks like there's nothing about this class that is specific to `sdxl`, right? Seems like this class isn't actually ever doing the `diffusers` load? It's not clear to me how this is actually getting a model into memory anywhere.